tests/integration.rs

```diff
@@ -3502,4 +3502,48 @@ mod tests {
             Ok(())
         });
     }
+
+    #[cfg_attr(not(feature = "mmio"), test)]
+    fn test_initramfs() {
+        test_block!(tb, "", {
+            let mut clear = ClearDiskConfig::new();
+            let guest = Guest::new(&mut clear);
+            let mut workload_path = dirs::home_dir().unwrap();
+            workload_path.push("workloads");
+
+            let mut kernel_path = workload_path.clone();
+            kernel_path.push("vmlinux");
+
+            let mut initramfs_path = workload_path;
+            initramfs_path.push("alpine_initramfs.img");
+
+            let test_string = String::from("axz34i9rylotd8n50wbv6kcj7f2qushme1pg");
+            let cmdline = format!("console=hvc0 quiet TEST_STRING={}", test_string);
+
+            let mut child = GuestCommand::new(&guest)
+                .args(&["--kernel", kernel_path.to_str().unwrap()])
+                .args(&["--initramfs", initramfs_path.to_str().unwrap()])
+                .args(&["--cmdline", &cmdline])
+                .spawn()
+                .unwrap();
+
+            thread::sleep(std::time::Duration::new(20, 0));
```

Comment:
this VM boots and prints in around 0.5s, maybe I should lower the sleep here to 5s?